src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -161,6 +161,35 @@ describe('<DateSelector />', () => {
     });
   });
 
+  describe('timezones adjustments', () => {
+    const date = '2019-01-01';
+    let getTimezoneOffset: jest.SpyInstance;
+
+    beforeEach(() => {
+      getTimezoneOffset = jest.spyOn(Date.prototype, 'getTimezoneOffset');
+    });
+
+    afterEach(() => {
+      getTimezoneOffset.mockRestore();
+    });
+
+    it(`sets the correct values for ${date} with timezone offset -540`, () => {
```

Comment:
No need to interpolate here. 
It could just be `sets the correct selected date on DatePicker for date with timezone offset`?